TownSuite.Web.ImageGen/Dockerfile

```diff
@@ -5,6 +5,26 @@ WORKDIR /app
 EXPOSE 80
 EXPOSE 443
 
+FROM mcr.microsoft.com/dotnet/sdk:6.0-alpine AS libheif-build
+RUN apk add --no-cache build-base git cmake yasm perl libde265 libde265-dev x265 x265-dev
+WORKDIR /tmp
+RUN git clone --depth 1 https://aomedia.googlesource.com/aom
+WORKDIR /tmp/aom
+RUN mkdir -p build && cd build && cmake .. && make -j$(nproc) && make install
+WORKDIR /app
+RUN git clone --depth 1 https://github.com/strukturag/libheif.git
+WORKDIR /app/libheif
+RUN mkdir build
+WORKDIR /app/libheif/build
+RUN cmake \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DWITH_EXAMPLES=OFF \
+    -DWITH_LIBDE265=ON \
+    -DWITH_X265=ON \
+    ..
+RUN make -j$(nproc)
+RUN make install
+

```

Comment:
Had to do much more building from source than preferred. 
I would like to skip building aom but ```apk add aom``` installs but is not detected when building libheif. Ill do some more experimentation with this.
Not sure if we can avoid building libheif.